metadata/namespaces.go

```diff
@@ -130,6 +131,14 @@ func (s *namespaceStore) List(ctx context.Context) ([]string, error) {
 	return namespaces, nil
 }
 
+func englishJoin(xs []string) string {
```

Comment:
I would rather keep things simple and just use `strings.Join` here
for same reasons discussed in: https://github.com/containerd/containerd/pull/4248#issuecomment-628668039

Reply:
Sounds good. Will do.